scripts/prepare/index.ts

```diff
@@ -92,7 +106,7 @@ async function getPackageJsonWithUpdates(file: string): Promise<PackageJson> {
   return json;
 }
 
-const rootPackageJson = await getPackageJsonWithUpdates('package.json');
+rootPackageJson = await getPackageJsonWithUpdates('package.json');
```

Comment:
Minor: Could the `'package.json'` be defined as the default value of the arg in `getPackageJsonWithUpdates`?
```suggestion
rootPackageJson = await getPackageJsonWithUpdates('package.json');
```